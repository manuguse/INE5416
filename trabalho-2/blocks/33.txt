1 1 1 6 6 12 12 12 12 12
1 1 6 6 7 13 18 18 12 21
1 6 6 6 7 13 13 18 18 21
2 2 7 7 7 13 13 14 14 21
3 3 3 7 7 14 14 14 21 21
3 3 3 10 10 15 16 16 19 19
4 3 8 11 11 11 16 19 19 19
4 5 8 11 9 9 16 20 19 19
5 5 9 9 9 16 16 20 20 20
5 5 5 9 9 17 17 20 20 20