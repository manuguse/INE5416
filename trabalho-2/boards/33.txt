- 6 - - 3 - 4 5 6 -
4 - - 6 - - - - 1 -
2 - - 2 - - 5 2 - -
- 2 - - 5 1 - - 4 3
2 6 - 2 3 - 2 - - 1
- - 5 - 2 - - 5 6 -
- - - - - 2 - 1 4 -
- - - - 7 - - - 2 -
6 - 2 4 - 2 - - - 5
2 - 1 - 5 - - 1 6 3