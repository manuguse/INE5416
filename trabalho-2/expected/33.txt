5 6 1 7 3 2 4 5 6 3
4 3 5 6 7 3 1 4 1 5
2 1 4 2 6 2 5 2 3 4
1 2 1 4 5 1 4 3 4 3
2 6 7 2 3 5 2 1 2 1
1 4 5 1 2 1 6 5 6 7
2 3 2 4 1 2 4 1 4 5
1 5 1 3 7 1 3 4 2 3
6 4 2 4 6 2 1 2 7 5
2 3 1 3 5 1 2 1 6 3